src/convert.cpp

```diff
@@ -12,15 +12,24 @@
 
 #include "src/convert.h"
 #include "src/log.h"
+#include "src/jsset.h"
 
-
+ddwaf_object* to_ddwaf_object(
+        ddwaf_object *object,
+        Napi::Env env,
+        Napi::Value val,
+        int depth,
+        bool lim,
+        JsSet jsSet
+);
 
```

Comment:
doesn't this go in convert.h ?

Reply:
This object is not available from other files, so not need to be in `convert.h` file. But in c++, the methods order matters, so we need to have it defined before call to it, and can't set the implemented method at the beginning, because it is calling to other internal methods.

we don't have this problem before, because the method without `JsSet` is defined in the `convert.h` file.